guides/common/modules/proc_upgrading-a-disconnected-project-server.adoc

```diff
@@ -44,107 +44,135 @@ This prevents the upgrade process from overwriting your DNS and DHCP configurati
 ----
 # rm /etc/yum.repos.d/*
 ----
-. Obtain the latest ISO files by following the {InstallingServerDisconnectedDocURL}downloading-the-binary-dvd-images_satellite[Downloading the Binary DVD Images] procedure in _{InstallingServerDisconnectedDocTitle}_.
 
-. Create directories to serve as a mount point, mount the ISO images, and configure the `rhel8` repository by following the {InstallingServerDisconnectedDocURL}configuring-the-base-operating-system-with-offline-repositories_satellite[Configuring the base operating system with offline repositories] procedure in _{InstallingServerDisconnectedDocTitle}_.
+.Procedure on the connected {ProjectServer}
+. Ensure that the following repositories are synchronized on your connected {ProjectServer}:
```

Comment:
This would introduce a custom block title (`.Procedure on the connected...`) and those are not supported for the DITA migration. It's better not to introduce new ones. My suggestion for an alternative:

```suggestion
. Export the required repositories from your connected {ProjectServer} to your disconnected {ProjectServer}:
.. Ensure that the following repositories are synchronized on your connected server:
```